src/components/TokenEnterAmount.tsx

```diff
@@ -465,8 +467,6 @@ export default function TokenEnterAmount({
 
 const styles = StyleSheet.create({
   rowContainer: {
-    borderWidth: 1,
-    borderColor: Colors.blue200,
```

Comment:
if you want to keep the diff minimal, you can leave the color here, and only set botterBottomWidth above conditionally